src/Data/List/Relation/Binary/Subset/Propositional/Properties.agda

```diff
@@ -231,6 +231,10 @@ module _ {P : Pred A p} (P? : Decidable P) where
   filter-⊆ : ∀ xs → filter P? xs ⊆ xs
   filter-⊆ = Setoidₚ.filter-⊆ (setoid A) P?
 
+  module _ {Q : Pred A q} (Q? : Decidable Q) where
+
+    filter⁺′ : P ⋐ Q → ∀ {xs ys} → xs ⊆ ys → filter P? xs ⊆ filter Q? ys
+    filter⁺′ = Setoidₚ.filter⁺′ (setoid A) P? (resp P) Q? (resp Q)
```

Comment:
Is there any reason not to call this one `filter⁺` and shadow the setoid
version [like we did for `map⁺`](http://agda.github.io/agda-stdlib/Data.List.Relation.Binary.Sublist.Propositional.Properties.html#1230)?

Reply:
The same reason why we don't call the setoid version `filter⁺`. There is a lemma in this file that incorrectly had that name before. Currently that name is deprecated. Can't use the name `filter⁺` without breaking backwards compatibility...